splunk_eventgen/lib/plugins/output/syslogout.py

```diff
@@ -26,6 +34,8 @@ def __init__(self, sample, output_counter=None):
 
         loggerName = 'syslog' + sample.name
         self._l = logging.getLogger(loggerName)
+        if syslogAddHeader:
+            self._l.addFilter(HostFilter(host=sample.host))
```

Comment:
please update the host setting in configuration docs in [eventgen.conf.spec](https://github.com/splunk/eventgen/blob/c48874703a09756847b3de91eaac2e6b49c8a381/splunk_eventgen/splunk_app/README/eventgen.conf.spec#L223)
Because now, host is not only used in splunkstream mode.